File: app/src/main/java/com/xattacker/android/bingo/LastViewRemover.java
package com.xattacker.android.bingo;

import android.graphics.Color;
import android.os.Handler;
import android.view.View;
import android.view.animation.Animation;
import android.view.animation.Animation.AnimationListener;

public class LastViewRemover implements AnimationListener
{
	private View _lastView;
	private int _count;
	
	public LastViewRemover(View aView)
	{
		_lastView = aView;
		_count = 0;
	}

	public void onAnimationStart(Animation arg0)
	{
		if (_lastView != null)
		{
			_lastView.setBackgroundColor(Color.RED);
		}
	}
	
	public void onAnimationEnd(final Animation arg0)
	{
		if (_lastView != null)
		{
			_lastView.setBackgroundColor(Color.TRANSPARENT);
		}
		
		
		_count++;
		
		if (_count < 2)
		{
			// repeat 2 times
			Handler handler = new Handler();
			handler.postDelayed
			(
			new Runnable() 
			{
			  public void run() 
			  {	
				  _lastView.startAnimation(arg0);
			  }
			},
			300
			);
		}
	}
	
	public void onAnimationRepeat(Animation arg0)
	{
	}
}

File: app/src/main/java/com/xattacker/android/bingo/view/LastViewRemover.kt
package com.xattacker.android.bingo.view

import android.graphics.Color
import android.os.Handler
import android.view.View
import android.view.animation.Animation
import android.view.animation.Animation.AnimationListener

class LastViewRemover(private val _lastView: View?) : AnimationListener
{
    private var _count: Int = 0

    init
    {
        _count = 0
    }

    override fun onAnimationStart(arg0: Animation)
    {
        _lastView?.setBackgroundColor(Color.RED)
    }

    override fun onAnimationEnd(arg0: Animation)
    {
        _lastView?.setBackgroundColor(Color.TRANSPARENT)


        _count++

        if (_count < 2)
        {
            // repeat 2 times
            val handler = Handler()
            handler.postDelayed({_lastView!!.startAnimation(arg0)}, 300)
        }
    }

    override fun onAnimationRepeat(arg0: Animation)
    {
    }
}
